.pipelines/pipeline.user.windows.yml

```diff
@@ -1,72 +1,72 @@
-environment:
-  host:
-    os: 'windows'
-    flavor: 'server'
-    version: '2019'
-  runtime:
-    provider: 'appcontainer'
-    image: 'cdpxwin1809.azurecr.io/global/vse2019:latest'
-    source_mode: 'link'
-versioning:
-  commands:
-    - !!defaultcommand
-      name: 'Generate Version'
-      command: '.scripts/version.cmd'
-# excluding our source code from being scanned for code sign validation
-signing_options:
-  profile: 'none'
-  codesign_validation_glob_pattern: '-:regex|\.(ps1|cmd|psm1)$;-|.gdn\\**'
-static_analysis_options:
-  policheck_options:
-    disable_tool_scan: false
-    fail_on_error: true
-restore:
-  commands:
-    - !!defaultcommand
-      name: 'Install Windows 10 SDK'
-      command: '.scripts/installWindows10Sdk.cmd'
-      logs:
-        - from: 'WinSdkInstall'
-          to: 'WinSdkInstall'
-          include:
-            - '**/*.log'
-    - !!defaultcommand
-      name: 'Install Java 11 JDK'
-      command: '.scripts/installJava11jdk.cmd'
-    - !!defaultcommand
-      name: 'Add OpenTelemetry to MavenLocal'
-      command: '.scripts/ot-gradle.cmd'
-      arguments: 'publishToMavenLocal --refresh-dependencies'
-    - !!defaultcommand
-      name: 'Clean and Resolve Dependencies'
-      command: '.scripts/gradle.cmd'
-      arguments: 'clean resolveDependencies --refresh-dependencies'
-    - !!defaultcommand
-      name: 'List Installation Information'
-      command: '.scripts/dumpInstallationInfo.cmd'
-test:
-  commands:
-    # - !!defaultcommand
-    #   name: 'Gradle Static Analysis'
-    #   command: '.scripts/gradle.cmd'
-    #   arguments: 'check -x test'
-    #   logs:
-    #     - to: 'Checks Reports'
-    #       include:
-    #         - '**/build/reports/**/*'
-    #       exclude:
-    #         - '**/build/reports/tests/**/*'
-    - !!testcommand
-      name: 'ApplicationInsights-Java Unit Tests'
-      command: '.scripts/gradle.cmd'
-      arguments: 'test'
-      fail_on_stderr: false
-      testresults:
-        - title: ':Unit Test Results'
-          type: 'junit'
-          include:
-            - '**/build/test-results/test/**/*'
-      logs:
-        - to: 'Test Logs'
-          include:
-            - '**/build/reports/tests/**/*'
+environment:
+  host:
+    os: 'windows'
+    flavor: 'server'
+    version: '2019'
+  runtime:
+    provider: 'appcontainer'
+    image: 'cdpxwin1809.azurecr.io/global/vse2019:latest'
+    source_mode: 'link'
+versioning:
+  commands:
+    - !!defaultcommand
+      name: 'Generate Version'
+      command: '.scripts/version.cmd'
+# excluding our source code from being scanned for code sign validation
+signing_options:
+  profile: 'none'
+  codesign_validation_glob_pattern: '-:regex|\.(dll|exe|ps1|cmd|psm1)$;-|.gdn\\**'
```

Comment:
this has the only change.. not sure why it show such big diff which is not necessary.

Reply:
looks like white space changed, maybe line endings?